lib/ecmascript.ts

```diff
@@ -6128,6 +6343,8 @@ export function CompareISODate(y1: number, m1: number, d1: number, y2: number, m
   return 0;
 }
 
+// Not abstract operations from the spec
```

Comment:
Hm - maybe for ease of organization we should move all the prod polyfill helper fns to a new file so that rebasing doesn't have to get confusing as the upstream file grows?

Maybe for another PR.